app/src/main/java/org/simple/clinic/drugs/sync/PrescribedDrugPayload.kt

```diff
@@ -41,7 +42,13 @@ data class PrescribedDrugPayload(
     val updatedAt: Instant,
 
     @Json(name = "deleted_at")
-    val deletedAt: Instant?
+    val deletedAt: Instant?,
+
+    @Json(name = "frequency")
+    val frequency: MedicineFrequency?,
+
+    @Json(name = "duration_in_days")
+    val durationInDays: Int?
```

Comment:
I was under the impression that we were only going to store this in the teleconsult record. If we make this change here, how will it work with the regular edit medicines flow (from the patient summary) where there is no UI to change the frequency and duration?

Reply:
We have decided to move it to `PrescribedDrug` to avoid having any metadata about the prescribed drugs when sending record and just send the prescribed drug UUID. You can find the conversation [here](https://simpledotorg.slack.com/archives/CV9V1DWLV/p1597820621045100?thread_ts=1597730690.017500&cid=CV9V1DWLV). As for the medicines UI, for now, we are continuing with the existing UI, but we might change it in future if needed.